### ISSUE TITLE
Errors with Installing React Native Sentry Cocoapod

### ISSUE THREAD

**Issue by thenewt15** [getsentry/react-native-sentry#349]
Installed for React Native with `npm install react-native-sentry --save`
Then ran `react-native link react-native-sentry`
This didn't add Sentry to my Pods folder, so I had to run `pod install`.

[I discovered the missing Pods when archiving failed for me].

During pods installation, a React folder was added, with a node_modules folder in it. This new node_modules folder is causing all kinds of errors during the packaging process because there are now two instance of several dependencies to resolve (because I already had a node_modules folder).

Additionally, its saying this React dependency (v0.11.0) is deprecated. 

Bottom line, I can't build my project anymore because of this.

**Comment by thenewt15** [comment]
Yes, I tried using that Podfile example. I finally got it working by removing the React subspecs in my Podfile that were already linked in XCode. I noticed in my 'Link Binary with Libraries' section that RCTImage, RCTText, RCTNetwork, and RCTWebSocket were already there. So I left them there and removed them from the Podfile. I think the double reference was causing issues.

**Comment by HazAT** [comment]
OK good to know, I will close this or now.
Feel free to reopen it.
